java-package/openbci_gui_helpers/src/main/java/openbci_gui_helpers/examples/TestNativeDiscovery.java

```diff
@@ -1,20 +1,17 @@
 package openbci_gui_helpers.examples;
 
-import java.util.Map;
-
 import openbci_gui_helpers.GUIHelper;
+import openbci_gui_helpers.GUIHelper.GanglionDevice;
 import openbci_gui_helpers.GanglionError;
 
-public class TestNativeDiscovery
-{
+public class TestNativeDiscovery {
 
-    public static void main (String[] args) throws GanglionError
-    {
-        Map<String, String> map = GUIHelper.scan_for_ganglions (3);
-        for (Map.Entry<String, String> entry : map.entrySet ())
-        {
-            System.out.println ("Key = " + entry.getKey () + ", Value = " + entry.getValue ());
+    public static void main(String[] args) throws GanglionError {
+        GanglionDevice[] devices = GUIHelper.scan_for_ganglions(3);
+        for (GanglionDevice device : devices) {
+            System.out.println("Identifier = " + device.identifier + ", Mac Address = " + device.mac_address
```

Comment:
Same here. Result of test?

Reply:
It passes for me, but it would be better if you also ran the test on your side so we can confirm it works on more than just my device.

Reply:
![image](https://github.com/OpenBCI/OpenBCI_GUI_Helpers/assets/84428015/57b0554b-9c6a-4804-b19d-980154d9de71)

Reply:
![image](https://github.com/OpenBCI/OpenBCI_GUI_Helpers/assets/84428015/fa934b25-f71d-4b4a-8ae3-6f18fb9601bb)